sagemaker-core/spark-processing-sm-core.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<p>Can you add a cell to delete the resources created in this notebook at the end?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
processing jobs once created can't be deleted. Once they are completed, costs stop getting incurred.